docs/topics/media-pipeline.rst

```diff
@@ -332,6 +346,23 @@ To handle media redirections, set this setting to ``True``::
 
     MEDIA_ALLOW_REDIRECTS = True
 
+.. _topics-media-pipeline-cache:
+
+Limiting cache size
+-------------------
+
+.. setting:: MEDIA_CACHE_SIZE
+
+By default media pipeline caches meta information
+(url, path and checksum) for all downloaded files and images.
+This might consume noticeable amount of memory for large-scale crawls
+with millions of items downloaded. Set it to the number of most recently
+used items you wish to keep::
```

Comment:
Does cache size depend on a number of items, or on a number of files uploaded/downloaded?